tpcx_bb/queries/q04/tpcx_bb_query_04_sql.py

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, BlazingSQL, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
+from xbb_tools.cluster_startup import attach_to_cluster
+from xbb_tools.sessionization import get_sessions
+
+import cudf
```

Comment:
Can we push this down to local scope inside a function?

Reply:
Updated.